demos/cc-gramine-demo/README.md

```diff
@@ -0,0 +1,94 @@
+# Introduction
+
+This demo presents a _privacy-preserving machine learning_ (PPML) application
+use case built using Gramine-SGX Library OS and PyTorch. It follows the
+Gramine's [PyTorch PPML Framework Tutorial](https://gramine.readthedocs.io/en/latest/tutorials/pytorch/index.html)
+but moves it to a Kubernetes orchestrated Cloud environment where the
+application is run as a confidential container process inside an Intel
+SGX enclave on an _untrusted machine_.
+
+In the demo, both input data and the machine learning model are encrypted
+inside the container and decrypted inside an Intel SGX based Trusted Execution
+Environment(TEE) using a key delivered from a trusted host after a successfull
+remote attestation. It shows how input data confidentiality as well as model's
+IP can be guaranteed in an untrusted environment.
+
+The high-level diagram (image credits: Gramine project) is as follows (best viewed using Github's light theme):
```

Comment:
Nit: Might be worth making "Gramine project" a link here.

Reply:
added link